sdk/resourcemanager/Proto.Client/compute/Placeholder/AvailabilitySetData.cs

```diff
@@ -52,13 +52,13 @@ public Azure.ResourceManager.Compute.Models.Sku Sku
         }
 
         /// <summary> A list of references to all virtual machines in the availability set. </summary>
-        public IList<SubResource> VirtualMachines
+        public IList<Azure.ResourceManager.Compute.Models.SubResource> VirtualMachines
```

Comment:
After introducing the general `SubResource` model it became ambigous in the proto code (as it is also referencing current track2 library), so I used full namespace.